jsk_rqt_plugins/src/jsk_rqt_plugins/plot_2d.py

```diff
@@ -195,9 +252,17 @@ def plot_one(self, msg, axes):
             model_ransac.fit(X, Y)
             line_X = X
             line_y_ransac = model_ransac.predict(line_X)
-            axes.plot(line_X, line_y_ransac, "r--",
-                      label="{0} x + {1}".format(model_ransac.estimator_.coef_[0][0],
-                                                 model_ransac.estimator_.intercept_[0]))
+            if len(model_ransac.estimator_.coef_) == 1:
+                coef = model_ransac.estimator_.coef_[0]
+            else:
+                coef = model_ransac.estimator_.coef_[0][0]
+            if not isinstance(model_ransac.estimator_.intercept_, list):
+                intercept = model_ransac.estimator_.intercept_
+            else:
+                intercept = model_ransac.estimator_.intercept_
```

Comment:
This is probably `intercept = model_ransac.estimator_.intercept_[0]` because it is `list`.
Sorry, this is my fault.